api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpHTSClientCardDataExportReport.java

```diff
@@ -0,0 +1,198 @@
+package org.openmrs.module.ugandaemrreports.reports;
+
+import org.openmrs.Concept;
+import org.openmrs.PatientIdentifierType;
+import org.openmrs.PersonAttributeType;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.metadatadeploy.MetadataUtils;
+import org.openmrs.module.reporting.cohort.definition.CohortDefinition;
+import org.openmrs.module.reporting.data.DataDefinition;
+import org.openmrs.module.reporting.data.converter.AgeConverter;
+import org.openmrs.module.reporting.data.converter.DataConverter;
+import org.openmrs.module.reporting.data.converter.ObjectFormatter;
+import org.openmrs.module.reporting.data.patient.definition.ConvertedPatientDataDefinition;
+import org.openmrs.module.reporting.data.patient.definition.PatientIdentifierDataDefinition;
+import org.openmrs.module.reporting.data.patient.library.BuiltInPatientDataLibrary;
+import org.openmrs.module.reporting.data.person.definition.AgeDataDefinition;
+import org.openmrs.module.reporting.data.person.definition.GenderDataDefinition;
+import org.openmrs.module.reporting.data.person.definition.PersonAttributeDataDefinition;
+import org.openmrs.module.reporting.dataset.definition.DataSetDefinition;
+import org.openmrs.module.reporting.dataset.definition.PatientDataSetDefinition;
+import org.openmrs.module.reporting.evaluation.parameter.Mapped;
+import org.openmrs.module.reporting.evaluation.parameter.Parameter;
+import org.openmrs.module.reporting.report.ReportDesign;
+import org.openmrs.module.reporting.report.definition.ReportDefinition;
+import org.openmrs.module.ugandaemrreports.data.converter.ObsDataConverter;
+import org.openmrs.module.ugandaemrreports.data.converter.PersonAttributeDataConverter;
+import org.openmrs.module.ugandaemrreports.definition.dataset.definition.NameOfHealthUnitDatasetDefinition;
+import org.openmrs.module.ugandaemrreports.library.DataFactory;
+import org.openmrs.module.ugandaemrreports.reporting.dataset.definition.SharedDataDefintion;
+import org.openmrs.module.ugandaemrreports.reporting.metadata.Dictionary;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Health Unit TB Register Report
+ * */
+
+@Component
+public class SetUpHTSClientCardDataExportReport extends UgandaEMRDataExportManager {
+
+
+	@Autowired
+	private DataFactory df;
+
+	@Autowired
+	SharedDataDefintion sdd;
+
+	@Autowired
+	private BuiltInPatientDataLibrary builtInPatientData;
+	
+	@Override
+	public String getDescription() {
+		return "HTS Client Card Data Export" ;
+	}
+	
+	@Override
+	public String getName() {
+		return " HTS Client Card Data Export";
+	}
+	
+	@Override
+	public String getUuid() {
+		return "96e0926d-1606-4de6-943f-cb036bdc15ad";
+	}
+	
+	@Override
+	public String getVersion() {
+		return "0.1.6";
```

Comment:
@dbaluku Start with 3.0.0 for all new HMIS reports

Reply:
Done